### PR TITLE
add AnalyticsService Wrapper #2562

### LINKED ISSUE #2562: add AnalyticsService Wrapper

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Closes  #2562

### CODE REVIEW

**Review by hboon — COMMENTED** [review]
@vladyslav-iosdev I missed the context for this. Could you summarise?

**Comment by vladyslav-iosdev** [comment]
That is just abstraction layer for analytics, For future in case if we would like to add more analytic services)

**Comment by hboon** [comment]
Great idea. Thanks :)

File: AlphaWallet/Analytics/Coordinators/AnalyticsService.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    func application(open url: URL, options: [UIApplication.OpenURLOptionsKey: Any]) {
+
+    }
+
+    func application(didReceiveRemoteNotification userInfo: [AnyHashable: Any]) {
+
+    }
+
+    func log(navigation: AnalyticsNavigation, properties: [String: AnalyticsEventPropertyValue]?) {
+
```
Comment (hboon) [inline AlphaWallet/Analytics/Coordinators/AnalyticsService.swift:58]: Needs to delegate this to `mixpanelService` too.

File: AlphaWallet/Analytics/Models/AnalyticsEventPropertyValue.swift [hunk 2]
```diff
@@ -3,23 +3,51 @@
 import Foundation
 
 protocol AnalyticsEventPropertyValue {
+    var value: Any { get }
```
Comment (hboon) [inline AlphaWallet/Analytics/Models/AnalyticsEventPropertyValue.swift:6]: Can you try to replace the call to `convertParameterToSdkSpecificVersion(_:)` with `value`?

**Review by hboon — CHANGES_REQUESTED** [review]

**Comment by hboon** [comment]
Force pushed after rebase master

**Review by hboon — APPROVED** [review]

**Pull request merged**